database/truncate.go

```
@@ -0,0 +1,40 @@
+
+func (t Truncate) Execute() error {
+	if t.env.App.IsProduction() {
+		panic("Cannot truncate production environment")
+	}
+
+	tables := GetSchemaTables()
+
+	for i := len(tables) - 1; i >= 0; i-- {
+
+		if !isValidTable(tables[i]) {
+			return errors.New(fmt.Sprintf("Table '%s' does not exist", tables[i]))
```

golangci-lint (1.64.8)

S1028: should use fmt.Errorf(...) instead of errors.New(fmt.Sprintf(...))

(gosimple)